kartothek/core/index.py

```diff
@@ -132,6 +132,16 @@ def __repr__(self) -> str:
             class_=type(self).__name__, attrs=", ".join(repr_str)
         )
 
+    def observed_values(self) -> np.array:
+        """
+        Return an array of all observed values
+        """
+        return np.fromiter(
+            (self.normalize_value(self.dtype, x) for x in self.index_dct.keys()),
+            count=len(self.index_dct),
```

Comment:
redundant?

Reply:
What exactly? The fromiter or the count? I used the fromiter to somehow reduce memory footprint, the count is an optional kwarg which improves performance since Numpy can allocate all memory up front

Reply:
I meant the count, but never mind then